apps/src/templates/lessonOverview/ResourceList.jsx

```diff
@@ -98,6 +132,13 @@ export default class ResourceList extends Component {
           {')'}
         </span>
       )}
+      {this.isGDocsUrl(resource.url) && (
+        <DropdownButton text="Make a Copy" color="gray" size="inline">
```

Comment:
I added `Make a Copy`, but I think `PDF`, `Microsoft Office`, and `Google Docs` would skip i18n because they're brand names. let me know if that's wrong

Reply:
that's a good point -- i think you're right? it might be worth asking someone on @code-dot-org/foundations if there's a standard/preference here

Reply:
[slack thread](https://codedotorg.slack.com/archives/CFTFD6BPV/p1666284378110509) confirming we shouldn't translate these